main.go

```diff
@@ -703,20 +711,14 @@ is available in the $BITRISE_IDEDISTRIBUTION_LOGS_PATH environment variable`)
 			fail("Failed to create tmp dir, error: %s", err)
 		}
 
-		if appDSYM != "" {
-			if err := command.CopyDir(appDSYM, dsymDir, false); err != nil {
-				fail("Failed to copy (%s) -> (%s), error: %s", appDSYM, dsymDir, err)
-			}
+		if len(appDSYM) > 0 {
+			exportDSYMs(dsymDir, appDSYM)
```

Comment:
After `appDSYM` made to be an array, the exporting algorithm turned to be the same as below, so it made sense to extract it.

Reply:
You may find the related change in `go-xcode` [here](https://github.com/bitrise-io/go-xcode/pull/95).